Gemfile

```diff
@@ -13,9 +13,6 @@ gem 'jbuilder', '~> 2.0'
 
 gem 'dotenv'
 
-gem 'react-rails'
-gem 'browserify-rails'
```

Comment:
@tiagojsag Browserify was just used as a replacement for webpack (because we couldn't find a webpack version for rails) to compile the files from ES6 to ES5. As we'll be using ES5 and Backbone, we don't need this anymore.